dev/ci/pnpm-install-with-retry.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+MAX_DURATION_IN_SECONDS=600 # seconds
```

Comment:
Is there a reason git thinks this is a new file vs a renamed file? Did you delete the yarn one?